### PR TITLE
View on Block Explorer button

### PR DESCRIPTION
**Pull request by jkadamczyk**
Fixes APP-35

## What changed (plus any additional context for devs)

* Implemented a view on block explorer button for tx details sheet

### CODE REVIEW

File: src/screens/transaction-details/TransactionDetails.tsx [hunk 1]
```diff
@@ -11,15 +11,22 @@ import { View } from 'react-native';
...
+    transaction: RainbowTransaction;
+  };
 };
 
-export const TransactionDetails: React.FC = () => {
-  const route = useRoute();
+type Props = {
+  route: RouteProp<RouteParams, 'TransactionDetails'>;
+  navigation: StackNavigationProp<RouteParams, 'TransactionDetails'>;
+};
```
Comment (estrattonbailey) [inline src/screens/transaction-details/TransactionDetails.tsx:25]: Nice

**Review by estrattonbailey — APPROVED** [review]

File: src/screens/transaction-details/TransactionDetails.tsx [hunk 2]
```diff
@@ -11,15 +11,22 @@ import { View } from 'react-native';
...
+  TransactionDetails: {
+    transaction: RainbowTransaction;
+  };
 };
 
-export const TransactionDetails: React.FC = () => {
-  const route = useRoute();
+type Props = {
+  route: RouteProp<RouteParams, 'TransactionDetails'>;
+  navigation: StackNavigationProp<RouteParams, 'TransactionDetails'>;
```
Comment (benisgold) [inline src/screens/transaction-details/TransactionDetails.tsx:24]: maybe i am missing something but is the `navigation` attribute actually used?

File: src/screens/transaction-details/components/TransactionDetailsHashAndActionsSection.tsx [hunk 3]
```diff
@@ -3,23 +3,53 @@ import { SingleLineTransactionDetailsRow } from '@/screens/transaction-details/c
 import { TransactionDetailsDivider } from '@/screens/transaction-details/components/TransactionDetailsDivider';
 import { shortenTxHashString } from '@/screens/transaction-details/helpers/shortenTxHashString';
 import lang from 'i18n-js';
```
Comment (benisgold) [inline src/screens/transaction-details/components/TransactionDetailsHashAndActionsSection.tsx:5]: wanna switch to new `@/languages`?

**Review by benisgold — APPROVED** [review]

File: src/screens/transaction-details/components/TransactionDetailsHashAndActionsSection.tsx (hunk 3 above)
Comment (jkadamczyk) [reply src/screens/transaction-details/components/TransactionDetailsHashAndActionsSection.tsx]: yes!

File: src/screens/transaction-details/TransactionDetails.tsx (hunk 2 above)
Comment (jkadamczyk) [reply src/screens/transaction-details/TransactionDetails.tsx:25]: Fixed that!

**Pull request merged**